gradle/libs.versions.toml

```diff
@@ -163,6 +162,6 @@ xz = { module = "org.tukaani:xz", version.ref = "xz" }
 
 [bundles]
 exposed = ["exposedCore", "exposedDao", "exposedJdbc", "exposedJavaTime"]
-hoplite = ["hopliteCore", "hopliteHocon", "hopliteYaml"]
+hoplite = ["hopliteCore", "hopliteHocon"]
```

Comment:
Should this drop `"hopliteHocon"` instead?

Reply:
Yes, of course :)